resources/test/tests/functional/log-insertion.html

```diff
@@ -6,6 +6,9 @@
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
 <script>
+test(function(t) {
+  assert_equals(document.body, null);
+}, "Log insertion before load");
```

Comment:
I wonder if it's worthwhile having a second test to make sure completion of the first test doesn't trigger a body to be inserted?